app/Language.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Language extends Model
+{
+    //
+}
```

Comment:
Idk if a language needs to be stored as a normalized table like this. A string column for `locale` would be more than good enough on the tracks and lyrics tables.